### ISSUE TITLE
How to find the model with the best results?

### ISSUE THREAD

**Issue by kirit93** [automl/auto-sklearn#363]
When using autosklearn fit to get the best model for the data, ```show_models()``` returns ```MyDummyClassifier(configuration=1, init_params=None, random_state=None))``` as the model with the highest accuracy.

How do I find the type of classifier used and the hyperparameters for it?

**Comment by mfeurer** [comment]
I think the method `show_models()` should be renamed because it actually shows the final ensemble produced by Auto-sklearn. You can use the attribute `cv_results_` to see the actual model scores and the hyperparameters used. Keep in mind that the order in which the models are listed by `show_models()` does not give any information about their performance, only about the weight with which they are included in the final ensemble.

The fact that `MyDummyClassifier` is part of the ensemble is a bug (see #140) and will be fixed soon.

**Comment by mfeurer** [comment]
Closing this issue because of inactivity - please reopen if this issue is still present with the latest version.
